app/components/works/work_form_component.html.erb

```diff
@@ -1,8 +1,10 @@
 <div id="editor">
   <h1 class="row">Deposit your work</h1>
   <%= form_with model: work_form, url: [work_form.model.collection, work_form],
-      local: true,
-      data: { controller: 'progress' },
+      data: {
+        controller: 'progress',
+        action: 'ajax:error->progress#displayErrors'
```

Comment:
`ajax:error` and `ajax:success` are the hooks used by Rails (Turbolinks? Rails-UJS?). Use the error hook to display server-generated errors in the form without a page reload.